enhancements/api-review/azure_user_defined_tags.md

```diff
@@ -307,7 +307,12 @@ to 21 characters.
 - Gather feedback from the users.
 
 #### Tech Preview -> GA
-N/A. This feature is for Tech Preview, until decided for GA.
+- Feature was made available for TechPreviewOnly in OpenShift-4.13 release.
```

Comment:
I feel, details under this section should list criteria defined as in https://github.com/openshift/enhancements/blob/master/guidelines/enhancement_template.md#graduation-criteria